doc/06-distributed-monitoring.md

```diff
@@ -3001,7 +3001,7 @@ By default, the following features provide advanced HA functionality:
 * [Elasticsearch](09-object-types.md#objecttype-elasticsearchwriter)
 * [Gelf](09-object-types.md#objecttype-gelfwriter)
 * [Graphite](09-object-types.md#objecttype-graphitewriter)
-* [InfluxDB](09-object-types.md#objecttype-influxdbwriter)
+* [InfluxDB](09-object-types.md#objecttype-influxdb2writer)
```

Comment:
I think this should state that this applies to both versions since it links to a specific version.